src/containerbuddy/main.go

```diff
@@ -60,6 +62,14 @@ func main() {
 	select {}
 }
 
+func deregisterServices(config *Config) {
+	log.Println("Deregister All Services")
```

Comment:
Once we've got this closer to being merged we might want to consider cleaning up some of the log noise in these functions.